test/junit/scala/collection/immutable/LazyListLazinessTest.scala

```diff
@@ -853,7 +834,7 @@ class LazyListLazinessTest {
     }
     object CustomLong {
       import scala.language.implicitConversions
-      implicit def long2CustomInt(long: Long): CustomLong = CustomLong(long)
+      implicit def long2CustomLong(long: Long): CustomLong = CustomLong(long)
```

Comment:
doesn't really matter, but the class is called `CustomLong`, so we might as well be consistent